package.json

```diff
@@ -8,8 +8,19 @@
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev",
-    "clean": "npm run clean -ws && rm -f -r ./node_modules",
+    "---- TEST ----------------------------------------------------------": "",
     "test": "turbo run test",
+    "---- LINT ----------------------------------------------------------": "",
+    "lint:base": "eslint --ignore-path .gitignore --cache",
```

Comment:
looks a bit weird first, but it enables us to have the same config for all packages. Or else we would need to check if each individual script uses the correct ignore path, cache option etc.